dev-docs/bidders/markapp.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: bidder
+title: MarkApp
+description: MarkApp Bidder Adapter
+biddercode: markapp
+usp_supported: true
+schain_supported: true
+media_types: banner, video, native
+tcfeu_supported: false
+pbjs: true
+pbs: true
+pbs_app_supported: true
```

Comment:
Can you also include the additional meta fields here like
coppa_supported
dchain_supported
safeframes_ok
deals_supported
floors_supported
multiformat_supported
fpd_supported
...